site/src/pages/WorkspaceSettingsPage/WorkspaceSettingsForm.tsx

```diff
@@ -41,9 +45,20 @@ export const WorkspaceSettingsForm: FC<{
     error,
   );
 
+  const capitalizeFirstLetter = (inputString: string): string => {
+    return inputString.charAt(0).toUpperCase() + inputString.slice(1);
+  };
+
+  const autoUpdatesSet = ["never", "always"];
```

Comment:
could we define these at the top level so that they stay static instead of getting allocated/gc'd for each rerender?